packages/repo/bench/repo.bench.ts

```diff
@@ -23,9 +23,10 @@ describe('Repo Benchmarks', () => {
       if (i % 500 === 0) {
         console.log(i)
       }
-      await repo
-        .stageUpdate({
-          action: 'create',
+
+      await repo.applyCommit(
+        {
+          action: 'create' as const,
```

Comment:
It's small peas, but we could eventually consider getting an enum in here to skirt the need for `as const`.

Reply:
ended up just getting it in here 👌